### DISCUSSION TITLE
What are all of these old .junk files

### DISCUSSION THREAD

**Discussion by teajdowns** [datalust/seq-tickets#1648] [Help]
I see a number of old .junk files in the stream/ folder.  What are these and can they be deleted?

**Comment by liammclennan** [comment]
Hi @teajdowns ,

Those are temporary files created when something unusual happens to your Seq. If your Seq is running fine then you can safely delete them. If lots of them keep appearing then something is wrong and we will need to investigate further.

**Reply by teajdowns** [reply]
Ok, thank you.   I'll keep a closer eye on them.   I wasn't sure there is no documentation on what those file types are for that I could find so wanted to double check.

Thanks again.

**Reply by liammclennan** [reply]
You're welcome.
